script.js

```diff
@@ -0,0 +1,105 @@
+// DOM Elements
+const time = document.getElementById('time');
+const greeting = document.getElementById('greeting');
+const name = document.getElementById('name');
+const focus = document.getElementById('focus');
+const SHOW_AM_PM = true;
+const MIDNIGHT = 0;
+const NOON = 12;
+const ZERO = '0';
+const SIX_HOURS = 6;
+const EIGHTEEN_HOURS = 18;
+const TWENTY_THREE_HOURS = 23;
+
+// Show Time
+function showTime() {
+  const today = new Date();
+  let hour = today.getHours();
+  const min = today.getMinutes();
+  const sec = today.getSeconds(); 
+
+  // Set AM or PM
+  const amPm = hour >= NOON ? 'PM' : 'AM';
+
+  // 12hr Format
+  hour !== MIDNIGHT ?  hour % 12 || NOON : MIDNIGHT; 
+
+  // Output Time
+  time.innerHTML = `${hour}<span>:</span>${addZero(min)}<span>:</span>${addZero(sec)} ${SHOW_AM_PM ? amPm : ''}`;
+  setTimeout(showTime, 1000);
+}
+
+// Add Zeros
```

Comment:
Name of function explains the meaning. Remove the comment.

Reply:
DONE!